homeassistant/components/triggercmd/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for TRIGGERcmd integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import jwt
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN  # pylint:disable=unused-import
+from .hub import Hub
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({("token"): str})
+
+
+async def validate_input(hass: HomeAssistant, data: dict) -> dict[str, Any]:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from DATA_SCHEMA with values provided by the user.
+    """
+    if len(data["token"]) < 100:
+        raise InvalidToken
+
+    tokenData = jwt.decode(data["token"], options={"verify_signature": False})
+    if not tokenData["id"]:
+        raise InvalidToken
+
+    hub = Hub(hass, data["token"])
+    result = await hub.test_connection()
```

Comment:
The hub `test_connection` function just waits 1 second and returns `True` so this will never raise CannotConnect

Reply:
You're right.  I removed it.